manifests/emissary/emissary-defaultns.yaml.in

```diff
@@ -639,7 +637,7 @@ spec:
           value: https://app.getambassador.io/
         - name: AES_SNAPSHOT_URL
           value: http://emissary-ingress-admin.default:8005/snapshot-external
-        image: $imageRepo$:$version$
+        image: docker.io/ambassador/ambassador-agent:0.0.7
```

Comment:
Similar to what I mentioned above, I don't have an answer for this and I'm open to ideas. We will want to make sure that this continue's to get upgraded as  new releases of the agent are put out.